eng/build.sh

```diff
@@ -17,47 +17,74 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 usage()
 {
   echo "Common settings:"
-  echo "  --subset                   Build a subset, print available subsets with -subset help (short: -s)"
-  echo "  --os                       Build operating system: Windows_NT, Linux, FreeBSD, OSX, tvOS, iOS, Android, Browser, NetBSD or SunOS"
-  echo "  --arch                     Build platform: x86, x64, arm, armel, arm64 or wasm"
-  echo "  --configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c)"
-  echo "  --runtimeConfiguration     Runtime build configuration: Debug, Release or [CoreCLR]Checked (short: -rc)"
-  echo "  --librariesConfiguration   Libraries build configuration: Debug or Release (short: -lc)"
-  echo "  --projects <value>         Project or solution file(s) to build"
-  echo "  --verbosity                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
-  echo "  --binaryLog                Output binary log (short: -bl)"
-  echo "  --cross                    Optional argument to signify cross compilation"
-  echo "  --help                     Print help and exit (short: -h)"
+  echo "  --arch                          Build platform: x86, x64, arm, armel, arm64 or wasm."
```

Comment:
All feedback on build.cmd output applies to build.sh. In fact, would it be better to put this help text into a helptext.txt and have both of them dump it to console, instead of duplicating it?

Reply:
Oh, I see Unix uses `--` and Windows uses `-` .. that's unfortunate, oh well.

Reply:
The help text isn't identical between Windows and Unix. Unsure if it's worth extracting the identical pieces.

Reply:
Actually Unix also supports single `-`. We added support a while ago for our build scripts and arcade scripts to support same syntax cross platforms.

Reply:
I agree with @ViktorHofer here. I believe that even though Unix also accepts single `-`, the help text is different enough to make it arguably more confusing for future maintenance and consulting if we extract it.

Reply:
> Actually Unix also supports single -. We added support a while ago for our build scripts and arcade scripts to support same syntax cross platforms.

Perhaps it would make sense to make build.cmd support `--` and then just consistently document `--` for everything. (We could do the converse today, but `-` on Unix just makes us look like Windows people 😄 ) That ways combinations of flags could be copied and pasted between Unix and Windows.

Reply:
> Perhaps it would make sense to make build.cmd support -- and then just consistently document -- for everything. (We could do the converse today, but - on Unix just makes us look like Windows people 😄 ) That ways combinations of flags could be copied and pasted between Unix and Windows.

Good luck with fighting with Powershell to accept double `--`  😄 . I think we tried it and it was messy/not possible. (with messy I think we would need to add parsing to the build.cmd script and parse out the `--` and pass down `-` to powershell. 

Also, that means we would need to do this in all the repos, as we try to be consistent to have a unified experience in the dotnet/* world.

Reply:
Fair enough ...